src/test/java/io/shapelets/khiva/LibraryTest.java

```diff
@@ -67,7 +86,84 @@ public void testGetDevice() {
     }
 
     @Test
-    public void testGetKhivaVersion() {
-        Assert.assertEquals(Library.getKhivaVersion(), "0.1.0");
+    public void testGetKhivaVersion() { Assert.assertEquals(Library.getKhivaVersion(), getKhivaVersionFromGithub()); }
+
+    private String getKhivaVersionFromGithub() {
+        String response = getTagsFromGitHub();
+        Gson gson = new GsonBuilder().create();
+        GithubTag[] tagsArray = gson.fromJson(response, GithubTag[].class);
+        String version = tagsArray[tagsArray.length-1].getName();
+        version = version.replace("v", "");
+        version = version.replace("-RC", "");
+
+        return version;
+    }
+
+    private String getTagsFromGitHub() {
+        String response = "";
```

Comment:
This can be a StringBuffer and then on line 117 use response.append(). And on 128 response.toString()